MediaManager/Plugin.MediaManager.Abstractions/Implementations/MediaManagerBase.cs

```diff
@@ -255,7 +291,8 @@ private void OnPlayingChanged(object sender, PlayingChangedEventArgs e)
         {
             if (sender != CurrentPlaybackManager) return;
             MediaFinished?.Invoke(sender, e);
-            await PlayNext();
+            if (MediaQueue.Repeat == RepeatType.RepeatOne) await Seek(TimeSpan.Zero);
```

Comment:
If the user then manually click PlayNext(), should it then get repeated or just if the media finished?